c2c/template.py

```diff
@@ -282,8 +283,11 @@ def read_vars(vars_file):
                 if "cmd" in interpreter:
                     cmd = interpreter["cmd"][:]  # [:] to clone
                     cmd.append(expression)
+                    ignore_error = interpreter.get("ignore_error", False)
                     try:
-                        evaluated = check_output(cmd)
+                        evaluated = check_output(
+                            cmd, stderr=subprocess.PIPE if ignore_error else None
```

Comment:
If your cmd outputs more than 1kb to stderr, you gonna block for ever. Pipes have a limited buffer and, if not consumed, block until it is consumed.

If you don't care about the output, you should do instead:
with open(os.devnull, "w") as dev_null:
    check_output(cmd, stderr=dev_null)